src/main/java/org/lanternpowered/server/network/buffer/LanternByteBuffer.java

```diff
@@ -111,6 +111,7 @@ public ByteBuffer touch(Object hint) {
 
     @Override
     public LanternByteBuffer order(ByteOrder order) {
+        // TODO: Move from deprecated usage of .order()
```

Comment:
This should also be deprecated in the API as well, that's the only reason why this is still here. The new methods like `writeintLE`, etc. should also be added to the API when that happens.